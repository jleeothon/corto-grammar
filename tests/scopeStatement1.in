class Dog: Animal:: INDENT member name: string
DEDENT
