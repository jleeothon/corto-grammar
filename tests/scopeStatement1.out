(program (statement (scopeStatement (postscopeStatement (definition (declaration (typeLabel (identifier class)) Dog) : (initializerList (initializerElem (initializerValue (identifier Animal))))) :: INDENT (statement (definition (declaration (typeLabel (identifier member)) name) : (initializerList (initializerElem (initializerValue (identifier string))))) \n) DEDENT)) \n))
